src/azul/plugins/metadata/hca/transform.py

```diff
@@ -274,6 +274,23 @@ def from_index(self, value: str) -> Optional[JSON]:
 value_and_unit: ValueAndUnit = ValueAndUnit()
 
 
+class Nested(PassThrough[JSON]):
+    properties: Mapping[str, FieldType]
+
+    def __init__(self, **properties):
+        super().__init__(es_type='nested')
+        self.properties = properties
+
+    # This allows for some interoperability between fields that are Nested vs
+    # plain dictionaries. Currently this is only used in
+    # avro_pfb._entity_schema_recursive.
```

Comment:
Comments like this get stale in no time. I should be able to search for usages of a function with a tool like PyCharm and find references that way.

Try the `as_dict` approach we talked about in PL.